public/app/features/explore/TraceView/components/DetailsPanel/AccordianLogs.tsx

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) 2017 Uber Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { css } from '@emotion/css';
+import { sortBy as _sortBy } from 'lodash';
+import * as React from 'react';
+
+import { GrafanaTheme2 } from '@grafana/data';
+import { useStyles2 } from '@grafana/ui';
+
+import { autoColor } from '../Theme';
+import AccordianKeyValues from '../TraceTimelineViewer/SpanDetail/AccordianKeyValues';
+import { formatDuration } from '../TraceTimelineViewer/utils';
+import { TNil } from '../types';
+import { TraceLog, TraceKeyValuePair, TraceLink } from '../types/trace';
+import { ubMb1 } from '../uberUtilityStyles';
+
+const getStyles = (theme: GrafanaTheme2) => {
+  return {
+    AccordianLogsFooter: css`
+      label: AccordianLogsFooter;
+      color: ${autoColor(theme, '#999')};
+    `,
+  };
+};
+
+export type AccordianLogsProps = {
+  interactive?: boolean;
+  linksGetter: ((pairs: TraceKeyValuePair[], index: number) => TraceLink[]) | TNil;
+  logs: TraceLog[];
+  onItemToggle?: (log: TraceLog) => void;
+  openedItems?: Set<TraceLog>;
+  timestamp: number;
+};
+
+export default function AccordianLogs(props: AccordianLogsProps) {
```

Comment:
Mostly copy of original `AccordianLogs` component with design tweaks to suit card layout with tabs.